src/js/popup/HistoricCtrl.js

```diff
@@ -0,0 +1,39 @@
+(function() {
+  'use strict';
+
+  function HistoricCtrl($scope, $timeout, RatesService, chrome) {
+    var vm = this;
+
+    vm.loaded = false;
+
+    RatesService.getHistoric(updateDisplayedRates);
+
+    /**
+     * Updates displayed historic rates.
+     * Prepares view vars for the graphic. Gdo
+     *
+     * @param  {Array} rates Array of rates through history
+     * @return {undefined}
+     */
+    function updateDisplayedRates(rates) {
+      vm.labels = [];
+      vm.series = ['Compra', 'Venta'];
+      vm.data = [[], []];
+
+      for(var i = rates.length - 1; i >= 0,  i >= rates.length - 20; i--) {
```

Comment:
Using comma, the evaluation corresponds to the last expression. Should use `&&`.